oct_converter/readers/binary_structs/fds_binary.py

```diff
@@ -1,88 +1,192 @@
-        param_scan_04_header (obj:Struct) : Defines param scan header
-        img_trc_02_header (obj:Struct) : Defines img trc header
-        param_obs_02_header (obj:Struct) : Defines param obs header
-        img_mot_comp_03_header (obj:Struct) : Defines img mot comp header
-        effective_scan_range_header (obj:Struct) : Defines effective scan range header
-        regist_info_header (obj:Struct) : Defines regist info header
-        result_cornea_curve_header (obj:Struct) : Defines result cornea curve header
-        result_cornea_thickness_header (obj:Struct) : Defines result cornea thickness header
-        contour_info_header (obj:Struct) : Defines contour info header
-        align_info_header (obj:Struct) : Defines align info header
-        fast_q2_info_header (obj:Struct) : Defines fast q2 info header
-        gla_littmann_01_header (obj : Struct) : Defines gla littmann 01 header
+        header (obj:Struct) : Defines structure of volume's header.
+        oct_header (obj:Struct) : Defines structure of OCT header (IMG_SCAN_03).
+        oct_header_2 (obj:Struct) : Defines structure of OCT header (IMG_SCAN_02).
+        fundus_header (obj:Struct): Defines structure of fundus header (IMG_OBS).
+        param_scan_04_header (obj:Struct) : Defines PARAM_SCAN_04 header.
+        param_scan_02_header (obj:Struct) : Defines PARAM_SCAN_02 header.
+        hw_info_03_header (obj:Struct) : Defines HW_INFO_03 header.
+        hw_info_02_header (obj:Struct) : Defines HW_INFO_02 header.
+        hw_info_01_header (obj:Struct) : Defines HW_INFO_01 header.
+        patient_info_02_header (obj:Struct) : Defines PATIENT_INFO_02 header.
+        file_info_header (obj:Struct) : Defines FILE_INFO header.
+        capture_info_02_header (obj:Struct) : Defines CAPTURE_INFO_02 header.
+        capture_info_header (obj:Struct) : Defines CAPTURE_INFO header.
+        img_trc_02_header (obj:Struct) : Defines IMG_TRC_02 header.
+        img_trc_header (obj:Struct) : Defines IMG_TRC header.
+        param_obs_02_header (obj:Struct) : Defines PARAM_OBS_02 header.
+        param_trc_02_header (obj:Struct) : Defines PARAM_TRC_02 header.
+        img_mot_comp_03_header (obj:Struct) : Defines IMG_MOT_COMP_03 header.
+        img_mot_comp_02_header (obj:Struct) : Defines IMG_MOT_COMP_02 header.
+        img_mot_comp_header (obj:Struct) : Defines IMG_MOT_COMP header.
+        img_projection_header (obj:Struct) : Defines IMG_PROJECTION header.
+        effective_scan_range_header (obj:Struct) : Defines EFFECTIVE_SCAN_RANGE header.
+        regist_info_header (obj:Struct) : Defines REGIST_INFO header.
+        regist_scan_02_header (obj:Struct) : Defines REGIST_SCAN_02 header.
+        result_cornea_curve_header (obj:Struct) : Defines RESULT_CORNEA_CURVE header.
+        result_cornea_thickness_header (obj:Struct) : Defines RESULT_CORNEA_THICKNESS header.
+        contour_info_header (obj:Struct) : Defines CONTOUR_INFO header.

```

Comment:
From the (old) information I have, CONTOUR_INFO and ALIGN_INFO are .fda only. I'm not sure if that's always true, and have gone back and forth on this very thought - leave it in, just in case someone *does* have that header present in an .fds, or assume that the old file information remains true and remove? If anything, if someone does raise an issue later that they've got an .fds with contours, it would be an easy fix to copy back in. I'm fine either way!

Reply:
i vote to remove them from .fds - seems like they're probably not there and, like you say, it would be easy to add back in if someone gives us reason to think they actually are.